lib/types.nix

```
@@ -402,7 +402,7 @@ rec {
       let list = addCheck (types.listOf elemType) (l: l != []);
       in list // {
         description = "non-empty " + list.description;
-        # Note: emptyValue is left as is, because another module may define an element.
+        emptyValue = { };
```

```suggestion
        emptyValue = { }; # no .value attr, meaning unset
```

nitpick: fight the misunderstanding that the empty value would be `{ }` here.

---

Good idea, pushed this change